tools/gblocks/gblocks.xml

```diff
@@ -0,0 +1,123 @@
+    #set $outputs_arr = []
+#end if
+#if $output_opt
+    #set $output_opt = $output_opt[:-1]
+#end if
+$output_opt
+> '$outputFile'
+    ]]></command>
+    <inputs>
+        <param name="inputFile" type="data" format="fasta,pir" label="Multiple Sequence Alignment" help="Multiple Sequence Alignment in Fasta format." />
+        <param argument="-t" type="select" label="Type Of Sequence">
+            <option value="p" selected="true">Protein</option>
+            <option value="d">DNA</option>
+            <option value="c">Codons</option>
+        </param>
+        <conditional name="advanced">
+            <param name="a" type="select" label="Specify Advanced Parameters">
+                <option value="-a">Yes</option>
+                <option value="" selected="true">No</option>
+            </param>
+            <when value="" />
+            <when value="-a">
+                <param argument="-b1" type="integer" min="1" value="" optional="true" label="Minimum number of sequences for a conserved position" help="Any integer bigger than half the number of sequences and smaller or equal than the total number of sequences" />
+                <param argument="-b2" type="integer" min="1" value="" optional="true" label="Minimum number of sequences for a flank position" help="Any integer equal or bigger than minimum number of sequences for a conserved position" />
+                <param argument="-b3" type="integer" value="8" label="Maximum number of contiguous nonconserved positions" />
+                <param argument="-b4" type="integer" value="10" min="2" label="Minimum length of a block" />
+                <param argument="-b5" type="select" label="Allowed gap positions">
+                    <option value="n" selected="true">None</option>
+                    <option value="h">With Half</option>
+                    <option value="a">All</option>
+                </param>
+                <param argument="-b6" type="select" label="Use similarity matrices">
+                    <option value="y" selected="true">Yes</option>
+                    <option value="n">No</option>
+                </param>
+                <param argument="-b0" type="integer" value="10" min="2" label="Minimum length of an intial block" />
+            </when>
+        </conditional> 
+        <param name="outputs" type="select" multiple="true" optional="true" display="checkboxes" label="Output formats">
+            <option value="-s" selected="true">Selected blocks</option>
+            <option value="-p" selected="true">Results and parameters File</option>
+            <option value="-n">Nonconserved blocks</option>
+            <option value="-u">Ungapped alignment</option>
+            <option value="-k">Mask file with the selected blocks</option>
+            <option value="-d">Postscript file with the selected blocks</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="outputFile" format="txt" label="${tool.name} on ${on_string}" />
```

Comment:
I think we could not save the tool output as a dataset (i.e. remove this line and line 36 above, plus the tests) and instead remove `optional="true"` for `outputs`.

Reply:
We should keep this file as it shows a summery of alignment quality, it will help users' who just want to know quality rather than looking at detailed alignment blocks